tests/lib/eslint.js

```diff
@@ -959,6 +959,21 @@ describe("eslint", function() {
             sinon.assert.calledTwice(spyLiteral);
             sinon.assert.calledOnce(spyBinaryExpression);
         });
+
+        it("SourceCode objects should be processed", function() {
+            var config = {
+                rules: {},
+                ecmaFeatures: {}
+            };
+
+            eslint.reset();
+            eslint.verify(code, config, filename, true);
+            var sourceCode = eslint.getSourceCode();
```

Comment:
This is a really roundabout way to get a `SourceCode` object, so it's not very clear what this test is doing. Can you just manually create it so you're not calling `verify()` twice?